trino.yaml

```diff
@@ -0,0 +1,165 @@
+package:
+  name: trino
+  version: 422
+  epoch: 0
+  description:
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - bash # some helper scripts use bash
+      - openjdk-17-default-jvm
+      - python3 # required by trino launcher script
+
+environment:
+  contents:
+    packages:
+      - wolfi-base
+      - busybox
+      - ca-certificates-bundle
+      - curl
+      - maven
+      - build-base
+      - openjdk-17
+      - git
+      - rsync
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/trinodb/trino.git
+      tag: ${{package.version}}
+      expected-commit: 2d1e9d7914234cd7b7af752571e66ced645618fc
+
+  - runs: |
+      export JAVA_HOME=/usr/lib/jvm/java-17-openjdk
+      export LANG=en_US.UTF-8
+      export INCLUDE="-I${JAVA_HOME}/include -I${JAVA_HOME}/include/linux"
+      export CFLAGS="-Wall -Werror -fPIC -shared ${INCLUDE}"
+
+      mkdir -p ${{targets.destdir}}/data/trino
+      mkdir -p ${{targets.destdir}}/etc/trino/catalog
+      mkdir -p ${{targets.destdir}}/usr/bin
+      mkdir -p ${{targets.destdir}}/usr/lib/trino/bin
+      mkdir -p ${{targets.destdir}}/usr/lib/trino/lib
+      mkdir -p ${{targets.destdir}}/usr/lib/trino/plugin
+      mkdir -p ${{targets.destdir}}/var/lib/trino
+      mkdir -p ${{targets.destdir}}/etc/security/
+      mkdir -p /home/build/trino-all
+      mkdir -p /home/build/plugins/default/
+      mkdir -p /home/build/plugins/optional/
+
+      git clone https://github.com/airlift/jvmkill.git /home/build/jvmkill
+      cd /home/build/jvmkill
+      gcc $CFLAGS -o libjvmkill.so jvmkill.c
+      chmod 644 libjvmkill.so
+
+      cd /home/build/
+      mvn package -q -DskipTests -T $(nproc) -pl ':trino-server'
+      tar -C /home/build/trino-all/ --strip-components 1 -xf core/trino-server/target/trino-server-${{package.version}}.tar.gz
+
+      mvn package -q -DskipTests -T $(nproc) -pl ':trino-cli'
+      chmod +x client/trino-cli/target/trino-cli-${{package.version}}-executable.jar
+
+      # rsync -av --force /home/build/trino-all/plugin/jmx /home/build/plugins/default/
+      # rsync -av --force /home/build/trino-all/plugin/memory /home/build/plugins/default/
+      # rsync -av --force /home/build/trino-all/plugin/tpcds /home/build/plugins/default/
+      # rsync -av --force /home/build/trino-all/plugin/tpch /home/build/plugins/default/
+      # rsync -av --exclude memory --exclude tpcds --exclude tpch --exclude jmx /home/build/trino-all/plugin/* /home/build/plugins/optional/
+      rsync -av /home/build/trino-all/plugin/* /home/build/plugins/optional/
```

Comment:
just curious why `rsync` instead of `cp`? do these come from some upstream build?

Reply:
to be honest, don't know much about that package, maybe @johnfosborneiii can answer that question, also I didn't find any `rsync` command anywhere on that project.

Reply:
I switched from mv/cp to rsync due to some "rename" errors that I was getting, I ended up resolving the root cause and it was unrelated to mv/cp so you could switch it back if that would be more consistent or preferred.